docs/docs/user-setup.md

```diff
@@ -88,34 +88,40 @@ powershell -Command "iex ((New-Object System.Net.WebClient).DownloadString('http
 </TabItem>
 </Tabs>
 
-### MCU Board Selection
+### Keyboard Selection
 
-When prompted, enter the number for the corresponding MCU board you would like to target:
+:::note
```

Comment:
How about moving this note box to below the `Keyboard Selection:` code block?

Reply:
I think this is a good idea, but what's the thought on more descriptive titles for the notes? I think once the note gets to be this long it makes it a little harder to parse, and having a more descriptive title might be helpful, but I don't know if this is done elsewhere in the docs.

Reply:
If I understand you correctly, yes we use titles for note boxes in a few places like here: https://zmk.dev/docs/behaviors/bluetooth#summary. I think we could add a name to this one as well.

Reply:
@dixls Are you planning to move this to the end of this subsection (and add a note title if you like)? I think that's the only comment I have left, otherwise looks good to me.

Reply:
Yes! Sorry, I had made these changes and then got pulled away before checking it over and pushing. Let me know how this version looks, not sure if the Note title feels redundant, but it may more successfully grab peoples attention.